### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by tangfayuan** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#127]
Hello, I would like to ask the code, is the loss function the sum of the losses on each picture in each batch? Are mask used to determine whether the key points are visible?
#
def l2_loss (input, target, task, batch_size):
 loss =(input -target)* mask
 loss =(loss *loss)//2/batch_size

 return loss.sum()
#

**Comment by Daniil-Osokin** [comment]
Hi, basically yes. Some images in COCO has unlabeled areas, mask is used to compute loss for labeled areas only.

**Comment by Daniil-Osokin** [comment]
Hope, it helped.
